src/phoenix/core/traces.py

```diff
@@ -246,7 +246,7 @@ def _consume_spans(self) -> None:
     def _process_span(self, span: pb.Span) -> None:
         span_id = SpanID(span.context.span_id)
         existing_span = self._spans.get(span_id)
-        if existing_span and existing_span.end_time:
+        if existing_span and existing_span.HasField("end_time"):
```

Comment:
When an `end_time` of `None` is pass to the protobuf span, it sets it to a default timestamp that is not truthy.

Reply:
Good catch!